inc/plugin.class.php

```diff
@@ -306,6 +310,16 @@ public function checkPluginState($directory) {
          }
 
          // Plugin is known but we are unable to load informations, it should be cleaned
+         if ($plugin->fields['state'] == self::TOBECLEANED) {
```

Comment:
I noticed that if a plugin was already "to be cleaned", it was updated on each global state check (on plugins page display).